src/main/java/org/jabref/Launcher.java

```diff
@@ -168,26 +168,29 @@ private static void initLogging(String[] args) {
     /**
      * @return true if JabRef should continue starting up, false if it should quit.
      */
-    private static boolean handleMultipleAppInstances(String[] args, RemotePreferences remotePreferences) {
+    private static boolean handleMultipleAppInstances(String[] args, RemotePreferences remotePreferences) throws InterruptedException {
         LOGGER.trace("Checking for remote handling...");
         if (remotePreferences.useRemoteServer()) {
             // Try to contact already running JabRef
             RemoteClient remoteClient = new RemoteClient(remotePreferences.getPort());
             if (remoteClient.ping()) {
                 LOGGER.debug("Pinging other instance succeeded.");
-                // We are not alone, there is already a server out there, send command line
-                // arguments to other instance
-                LOGGER.debug("Passing arguments passed on to running JabRef...");
-                if (remoteClient.sendCommandLineArguments(args)) {
-                    // So we assume it's all taken care of, and quit.
-                    // Output to both to the log and the screen. Therefore, we do not have an additional System.out.println.
-                    LOGGER.info("Arguments passed on to running JabRef instance. Shutting down.");
-                    return false;
+                // There is already a server out there, avoid showing log "Passing arguments" while no arguments are provided.
+                if (args.length == 0) {
+                    LOGGER.debug("This JabRef instance is already running. Please switch to that instance.");
```

Comment:
Higher log level to ensure that is really output on the console.

```suggestion
                    LOGGER.warn("This JabRef instance is already running. Please switch to that instance.");
```